### PR TITLE
Created home page + composant button

### PR DESCRIPTION
**Pull request by Bravax**

### CODE REVIEW

File: frontend/src/pages/Home/NavigationButton.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@import "../../variable.scss";
+
+.styleButton {
+  border: 0.15rem solid black;
+  border-radius: 0.5rem 3rem;
```
Comment (ItsMatDev) [inline frontend/src/pages/Home/NavigationButton.scss]: use px instead of rem for border

File: frontend/src/pages/Home/NavigationButton.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@import "../../variable.scss";
+
+.styleButton {
+  border: 0.15rem solid black;
```
Comment (ItsMatDev) [inline frontend/src/pages/Home/NavigationButton.scss]: px instead of rem

**Review by ItsMatDev — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Home/NavigationButton.scss (hunk 1 above)
Comment (Bravax) [reply frontend/src/pages/Home/NavigationButton.scss]: Amended

File: frontend/src/pages/Home/NavigationButton.scss (hunk 2 above)
Comment (Bravax) [reply frontend/src/pages/Home/NavigationButton.scss]: Amended

File: frontend/src/pages/Home/NavigationButton.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import "./NavigationButton.scss";
+
+function NavigationButton({ text, destination }) {
+  return (
+    <div className="styleButton">
```
Comment (ItsMatDev) [inline frontend/src/pages/Home/NavigationButton.jsx]: no div here, use button instead, see if link wrap button or button wrap link

File: frontend/src/pages/Home/NavigationButton.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  border-radius: 0.5rem 3rem;
+  box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+  margin-bottom: 3%;
```
Comment (ItsMatDev) [inline frontend/src/pages/Home/NavigationButton.scss:7]: use vh instead of %

File: frontend/src/pages/Home/NavigationButton.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@import "../../variable.scss";
+
+.styleButton {
+  border: 0.15rem solid black;
+  border-radius: 0.5rem 3rem;
+  box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
```
Comment (ItsMatDev) [inline frontend/src/pages/Home/NavigationButton.scss]: not good here but we will change it later with variable comming in an other PR

**Review by Yoann-Fortin — APPROVED** [review]

**Review by ItsMatDev — APPROVED** [review]

**Review by JulienOza — APPROVED** [review]

**Review by SebSamPro — APPROVED** [review]

**Pull request merged**